generator/templates/liquiditymining.payloaddeployment.template.ts

```diff
@@ -0,0 +1,27 @@
+export const liquidityMiningPayloadDeploymentTemplate = (
+  poolChain: string,
+  testContractName: string,
+  scriptName: string
+) => {
+  return`
+  // SPDX-License-Identifier: MIT
+  pragma solidity ^0.8.0;
+
+import {${testContractName}, IPermissionedPayloadsController}
+  from './${testContractName}.t.sol';
+import {${poolChain}Script} from '../../lib/aave-address-book/lib/aave-v3-origin/lib/solidity-utils/src/contracts/utils/ScriptUtils.sol';
+
```

Comment:
@Cycxyz I would put a comment too which has the command to run the script, so its bit easier